pyproject.toml

```diff
@@ -43,13 +43,15 @@ sphinx_rtd_theme = { version = "*", optional = true }
 sphinx-autoapi = { version = ">=3.0.0", optional = true }
 sphinx-math-dollar = { version = "*", optional = true }
 tox = { version = "*", optional = true }
+nbmake = { version = "*", optional = true } 
 
 # scripts dependencies.
 tqdm = { version = "*", optional = true }
 wandb = { version = "*", optional = true }
 scikit-learn = {version = "*", optional = true }
 scipy = { version = "*", optional = true }
 matplotlib = { version = "*", optional = true }
+torchtyping = { version = "*", optional = true }
```

Comment:
only comment - we should be able to fully remove this - if it's still in the notebooks, let's get rid of it.

Reply:
good point, done!